dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -227,24 +227,35 @@ Sound.prototype.newPlayableBufferSource = function(buffer, options) {
   var newSound = this.audioContext.createBufferSource();
 
   // Older versions of chrome call this createGainNode instead of createGain
-  var gainNode;
   if (this.audioContext.createGain) {
-    gainNode = this.audioContext.createGain();
+    this.gainNode = this.audioContext.createGain();
   } else if (this.audioContext.createGainNode) {
-    gainNode = this.audioContext.createGainNode();
+    this.gainNode = this.audioContext.createGainNode();
   } else {
     return null;
   }
 
   newSound.buffer = buffer;
   newSound.loop = !!options.loop;
-  newSound.connect(gainNode);
-  gainNode.connect(this.audioContext.destination);
-  gainNode.gain.value = typeof options.volume === "undefined" ? 1 : options.volume;
+  newSound.connect(this.gainNode);
+  this.gainNode.connect(this.audioContext.destination);
+  var startingVolume = typeof options.volume === "undefined" ? 1 : options.volume;
+  this.gainNode.gain.setValueAtTime(startingVolume, this.audioContext.currentTime);
```

Comment:
FWIW, MDN says that everything that supports [`AudioContext.createBufferSource()`](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/createBufferSource) which we're using in this method (fallback player doesn't hit this path) also supports [`AudioParam.setValueAtTime()`](https://developer.mozilla.org/en-US/docs/Web/API/AudioParam/setValueAtTime) except for Chrome 10-13, which we don't support.